examples/features/metadata/server/main.go

```diff
@@ -31,10 +31,9 @@ import (
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	pb "google.golang.org/grpc/examples/features/proto/echo"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
-
-	pb "google.golang.org/grpc/examples/features/proto/echo"
```

Comment:
Do we need this change?

Although we don't do it at all places, we try (as much as possible) to group the proto imports separately.

Reply:
Sorry for the typo, it was caused by the Goland. I will fix it.